src/Microsoft.ML/Runtime/EntryPoints/ImportTextData.cs

```diff
@@ -27,13 +27,23 @@ public sealed class Input
             public string CustomSchema = null;
         }
 
+        [TlcModule.EntryPointKind(typeof(ILearningPipelineLoader))]
+        public sealed class LoaderInput
+        {
+            [Argument(ArgumentType.Required, ShortName = "data", HelpText = "Location of the input file", SortOrder = 1)]
+            public IFileHandle InputFile;
+
+            [Argument(ArgumentType.Required, ShortName = "args", HelpText = "Arguments", SortOrder = 1)]
+            public TextLoader.Arguments Arguments = new TextLoader.Arguments();
+        }
+
         public sealed class Output
         {
             [TlcModule.Output(Desc = "The resulting data view", SortOrder = 1)]
             public IDataView Data;
         }
 
-        [TlcModule.EntryPoint(Name = "Data.TextLoader", Desc = "Import a dataset from a text file")]
+        [TlcModule.EntryPoint(Name = "Data.CustomTextLoader", Desc = "Import a dataset from a text file")]
```

Comment:
>CustomTextLoader [](start = 43, length = 16)

why we need it if we have generic version with strictly types?

Reply:
For the convenience text loader API.

---
In reply to: [186799880](https://github.com/dotnet/machinelearning/pull/38#discussion_r186799880) [](ancestors = 186799880)